edxval/tests/test_api.py

```diff
@@ -334,6 +334,43 @@ def test_get_videos_for_ids_duplicates(self):
         videos = list(api.get_videos_for_ids([edx_video_id, edx_video_id]))
         self.assertEqual(len(videos), 1)
 
+    def test_get_videos_for_ids_sort(self):
+        fish_id = constants.VIDEO_DICT_FISH["edx_video_id"]
+        star_id = constants.VIDEO_DICT_STAR["edx_video_id"]
+        other_id = "other-video"
+        Video.objects.create(**constants.VIDEO_DICT_STAR)
+        Video.objects.create(
+            client_video_id="other video",
+            duration=555.0,
+            edx_video_id=other_id
+        )
+
+        def check_sort(sort_field, asc_expected_ids):
+            """
+            Assert that sorting by given field returns videos in the expected
+            order (checking both ascending and descending)
+            """
+            def check_direction(sort_dir, expected_ids):
+                """Assert that the given videos match the expected ids"""
+                actual_videos = api.get_videos_for_ids(
+                    # Make sure it's not just returning the order given
+                    sorted(expected_ids),
```

Comment:
why sorted and not reversed? with sorted, it may still remain as the same order.